app/templates/macros/toolkit_forms.html

```diff
@@ -2,6 +2,7 @@
   {%
     with
     question=question_content.question,
+    hint=question_content.hint or None,
```

Comment:
`or ''`, but toolkit really should check for `{% if hint %}` instead of `if hint is defined`,  otherwise we'll get an empty hint span for all questions